public/assets/sass/components/_table.scss

```diff
@@ -0,0 +1,6 @@
+.expenses {
```

Comment:
I think this is not generic. When we add a new module like "users" module and list all users in a table. Should we add a "expenses" class to the users table?

Reply:
The style of expenses table is not generic, for now, it is only for it , when I will create another table, a common style will be created  and some edits will be done